tasks/main.yml

```diff
@@ -21,3 +21,42 @@
 - include: databases.yml
 - include: users.yml
 - include: replication.yml
+
+# Enable GTID and reconfigure
+- set_fact:
+    mysql_replication_gtid: true
+- name: Copy my.cnf global MySQL configuration.
+  template:
+    src: my.cnf.j2
+    dest: "{{ mysql_config_file }}"
+    owner: root
+    group: root
+    mode: 0644
+    force: "{{ overwrite_global_mycnf }}"
+
+- name: Restart mysql to load config.
+  service:
+    name: mysql
+    state: restarted
+
+- name: Stopslave.
+  mysql_replication:
+    mode: stopslave
+  when: >
+    ((slave.Is_Slave is defined and not slave.Is_Slave) or (slave.Is_Slave is not defined and slave|failed))
```

Comment:
@kaizoku This whole condition seems a bit long, no? If it's possible, it'd be great to simplify it somehow :) If not, then it's fine.

Reply:
I pulled that condition from the other one used in `replication.yml`.
I'm not sure of a good way to simplify it, otherwise it tries to stop the slave thread on the master, which of course has no slave thread.

Reply:
Allright, let's leave it like that then. Thanks for the comment ;)